pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/ingress_reconciler.go

```diff
@@ -453,6 +487,26 @@ func createIngress(isvc *v1beta1.InferenceService, useDefault bool, config *v1be
 	return desiredIngress
 }
 
+// getDomainList gets all the available domain names available with Knative Serving.
+func getDomainList(clientset kubernetes.Interface) *[]string {
+	res := new([]string)
+	ns := constants.DefaultNSKnativeServing
+	if namespace := os.Getenv(system.NamespaceEnvKey); namespace != "" {
+		ns = namespace
+	}
+
+	// Leverage the clientset to access the configMap to get all the available domain names
+	configMap, err := clientset.CoreV1().ConfigMaps(ns).Get(context.TODO(),
```

Comment:
You can use controllerRuntime client to get the configMap as well, just to be consistent with the rest of code in the kserve repo

Reply:
c.client does not work in terms of accessing the configmaps.